mapie/tests/test_common.py

```diff
@@ -1,19 +1,17 @@
 from inspect import signature
 from typing import Any, List, Tuple
 
-import pytest
 import numpy as np
+import pytest
+from mapie.classification import MapieClassifier
+from mapie.regression import MapieRegressor
 from sklearn.base import BaseEstimator
+from sklearn.exceptions import NotFittedError
 from sklearn.linear_model import LinearRegression, LogisticRegression
-from sklearn.pipeline import make_pipeline
 from sklearn.model_selection import KFold
-from sklearn.utils.validation import check_is_fitted
-from sklearn.exceptions import NotFittedError
+from sklearn.pipeline import make_pipeline
 from sklearn.utils.estimator_checks import parametrize_with_checks
-
-from mapie.regression import MapieRegressor
-from mapie.classification import MapieClassifier
-
+from sklearn.utils.validation import check_is_fitted
```

Comment:
Why changing the order of the imports ? It breaks the convention of internal packages (mapie) imported last.

If this is due to Black, please turn it off when contributing to open-source projects.

Reply:
it was indeed a option in .vscode.json